src/steamship/client/client.py

```diff
@@ -108,20 +124,52 @@ def embed_and_search(
             space=space,
         )
 
+    @staticmethod
     def use(
+        package_handle: str,
+        instance_handle: str = None,
+        config: Dict[str, Any] = None,
+        version: str = None,
+        reuse: bool = True,
+        **kwargs,
+    ) -> AppInstance:
+        """Creates/loads an instance of package `package_handle`.
+
+        The instance is named `instance_handle` and located in the Workspace named `instance_handle`.
+
+        For example, one may write the following to always get back the same package instance, no matter how many
+        times you run it, scoped into its own workspace:
+
+        ```python
+        plugin = Steamship.use('package-handle', 'instance-handle')
+        ```
+        """
+        kwargs["workspace"] = instance_handle
+        client = Steamship(**kwargs)
+        return client.use(
```

Comment:
I would personally use `_instance_use` here as this eases code navigation. Took me a while to understand/remember that use is actually pointing to `_instance_use`.

Reply:
Done